src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -261,3 +274,32 @@ To use the EKCO add-on to remove a node:
    The EKCO purge script For information about the EKCO purge script, see [Purge Nodes](/docs/add-ons/ekco#purge-nodes) in _EKCO Add-on_.
 
 1. Remove the node from the cluster.
+
+## Troubleshoot Node Removal
+
+This section includes information about troubleshooting issues with node removal in Rook Ceph clusters.
+
+### Rook Ceph v1.0.4 is Unhealthy with Mon Pods Not Rescheduled
+
+#### Symptom
+
+When you run `kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status` for a Rook Ceph v1.0.4 cluster, you see that Ceph is in an unhealthy state where failed mons are not rescheduled. 
+
+For example:
+
+```
+health: HEALTH_WARN
+        1/3 mons down, quorum a,c
+```
+
+#### Cause
+
+This is caused by a known issue in Rook Ceph v1.0.4 where failed mons are not removed during mon failover. For more information, see [Mon is never rescheduled](https://github.com/rook/rook/issues/2262#issuecomment-460898915) in the rook GitHub repository.
+
+#### Solution
+
+To resolve this issue, manually stop the Rook Ceph operator, remove the failed mon from the configmap, delete the failed mon, and then rescale the operator.  
```

Comment:
What about we add here the steps with the commands as we have in the community docs ? 
```suggestion
To resolve this issue you will need to perform the following steps: 
- Stop the Rook Operator by running: kubectl -n rook-ceph scale --replicas=0 deployment.apps/rook-ceph-operator
- Edit the configmap rook-ceph-mon-endpoints to (carefully) remove the failed mon from the list with the command kubect -n rook-ceph edit configmaps rook-ceph-mon-endpoints. You will see that this config map still with data info from the node that was purged. 
> ATTENTION: If you remove the wrong data endpoint it might cause unexpected behaviours as cause data loss.
- Delete the mon pod in Pending state. You can found its name by running kubectl -n rook-ceph get pod -l app=rook-ceph-mon and you can remove by running kubectl -n rook-ceph delete pod <mon-pod-name-in-pending-state>
- Then, rescale the Rook Operator by running: kubectl -n rook-ceph scale --replicas=1 deployment.apps/rook-ceph-operator
```

Reply:
Sounds good to me. Updated 👍